docker-compose.yml

```diff
@@ -29,7 +29,10 @@ services:
     env_file: 
       - ./.env
     volumes:
-      - ./backend:/var/www/app
+      - ./backend/routes:/var/www/app/routes
```

Comment:
If you could copy these lines, `git checkout master docker-compose.yml`, commit that, and then paste them back, that'd be great, as this is just a workaround until Docker for Mac maintainers issue a fix :)